### PR TITLE
Relationship between Istio and Envoy

### PR DESCRIPTION
**Pull request by zirain**
Please provide a description for what this PR is for.

fixes: https://github.com/istio/istio/issues/43086

### CODE REVIEW

**Review by jacob-delgado — APPROVED** [review]

File: content/en/docs/releases/supported-releases/index.md [hunk 1]
```diff
@@ -87,9 +86,22 @@ Please keep up-to-date and use a supported version.
...
+Istio's data plane related on [Envoy](https://github.com/envoyproxy/envoy).
+Istio cut major release branch after Envoy released, the relationship between two project summary as following:
+
+| Istio version | Envoy release |
+| ------------- | ------------- |
+| 1.17.x        | 1.25.x        |
+| 1.16.x        | 1.24.x        |
+| 1.15.x        | 1.23.x        |
+
+you can find Envoy commit message from [Istio/proxy](https://github.com/istio/proxy/blob/master/WORKSPACE#L38).
```
Comment (howardjohn) [inline content/en/docs/releases/supported-releases/index.md:107]:
```suggestion
In general, Istio releases tend to map one to one with Envoy releases. You can find the precise Envoy commit used in [istio/proxy](https://github.com/istio/proxy/blob/master/WORKSPACE#L38).
```

File: content/en/docs/releases/supported-releases/index.md [hunk 2]
```diff
@@ -87,9 +86,22 @@ Please keep up-to-date and use a supported version.
...
+| ---------------- | ---------------------------------------------------- |
 | 1.16.x           | 1.16.0+                                              |
 | 1.15.x           | 1.15.3+                                              |
 | 1.14.x           | 1.14.5 - End of life. A new CVE will NOT be patched. |
 | 1.13.x           | 1.13.9 - End of life. A new CVE will NOT be patched. |
 | 1.12 and earlier | None, all versions have known vulnerabilities.       |
+
+## Relationship between Istio and Envoy
+
+Istio's data plane related on [Envoy](https://github.com/envoyproxy/envoy).
```
Comment (howardjohn) [inline content/en/docs/releases/supported-releases/index.md:98]:
```suggestion
Istio's data plane is based on [Envoy](https://github.com/envoyproxy/envoy).
```

File: content/en/docs/releases/supported-releases/index.md [hunk 3]
```diff
@@ -87,9 +86,22 @@ Please keep up-to-date and use a supported version.
...
+Istio cut major release branch after Envoy released, the relationship between two project summary as following:
```
Comment (howardjohn) [inline content/en/docs/releases/supported-releases/index.md:99]:
```suggestion

The relationship between two projects versions:
```

**Review by howardjohn — APPROVED** [review]
LGTM, just minor style comments

File: content/en/docs/releases/supported-releases/index.md (hunk 1 above)
Comment (kfaseela) [inline content/en/docs/releases/supported-releases/index.md:107]:
```suggestion
The Envoy commit SHA used by Istio can be found at [Istio/proxy](https://github.com/istio/proxy/blob/master/WORKSPACE#L38).
```

File: content/en/docs/releases/supported-releases/index.md [hunk 4]
```diff
@@ -87,9 +86,23 @@ Please keep up-to-date and use a supported version.
...
+
+The relationship between two projects versions:
+
+| Istio version | Envoy version |
+| ------------- | ------------- |
+| 1.17.x        | 1.25.x        |
+| 1.16.x        | 1.24.x        |
+| 1.15.x        | 1.23.x        |
+
+In general, Istio releases tend to map one to one with Envoy releases. You can find the precise Envoy commit used in [istio/proxy](https://github.com/istio/proxy/blob/master/WORKSPACE#L38).
```
Comment (kfaseela) [inline content/en/docs/releases/supported-releases/index.md:108]:
```suggestion
In general, Istio releases tend to map one to one with Envoy releases. You can find the precise Envoy commit used by Istio in [istio/proxy](https://github.com/istio/proxy/blob/master/WORKSPACE#L38).
```

**Review by kfaseela — APPROVED** [review]

**Comment by kfaseela** [comment]
@ericvn : the document adds release 1.17 in the matrix, okay to cherry-pick to 1.16 ?

File: .spelling [hunk 5]
```diff
@@ -499,6 +503,7 @@ iptables
 IPv4
 IPv6
 Istio
+istio
```
Comment (ericvn) [inline .spelling:506]: We should not have added this to .spelling.

**Pull request merged**